pom.xml

```diff
@@ -237,10 +237,7 @@
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>2.10.4</version>
                 <configuration>
-                    <reportOutputDirectory>${project.reporting.outputDirectory}/apidocs
-                    </reportOutputDirectory>
-                    <destDir>apidocs</destDir>
-                    <nohelp>true</nohelp>
```

Comment:
I think this section was needed to ensure the javadoc ended in the right directory to make this easy to deploy to github.